src/content/quickstart/vsphere_aws/_index.md

```diff
@@ -105,31 +105,30 @@ We can overcome this limitation by using non-standard ports like 4501/UDP and 50
 Additionally, the default OpenShift deployments do not allow assigning an elastic public IP
 to existing worker nodes, something that's necessary at least on one end of the IPsec connections.
 
-To handle these requirements on AWS, we provide a script that will prepare your AWS OpenShift deployment for Submariner, and will create an
-additional gateway node with an external IP.
+To handle these requirements on AWS, we provide a script 
+[prep_for_subm.sh](https://github.com/submariner-io/submariner/blob/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh), 
+that will prepare your AWS OpenShift deployment for Submariner, and will create an additional gateway node with an external IP:
 
 ```bash
 
 curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
 chmod a+x ./prep_for_subm.sh
 
-./prep_for_subm.sh cluster-b      # respond yes when terraform asks
+./prep_for_subm.sh <ocp-install-path>      # respond yes when terraform asks to approve, or add after path: -auto-approve
```

Comment:
```suggestion
./prep_for_subm.sh cluster-b      # respond yes when terraform asks to approve, or add after path: -auto-approve
```

As this is a Quickstart guide and people might just want to copy/paste the commands for execution. In the examples above we use *cluster-b*, so lets use the same even for this command.